src/components/studies/app-design/LeadInvestigatorDropdown.tsx

```diff
@@ -107,43 +109,54 @@ const LeadInvestigatorDropdown: React.FunctionComponent<LeadInvestigatorDropdown
       setLeadInvestigatorOptions(leadInvestigatorArray)
     }
     getLeadResearchAccount()
-  }, [])
+  }, [token, orgMembership])
 
   return (
     <div>
       <Box marginLeft="8px">Lead Principle Investigator*</Box>
-      <div>
-        <button
-          className={clsx(
-            classes.selectPrincipleInvestigatorButton,
-            !isLeadInvestigatorDropdownOpen &&
-              classes.selectPrincipleInvestigatorButtonClosed,
-          )}
-          onClick={() => {
-            setIsLeadInvestigatorDropdownOpen(!isLeadInvestigatorDropdownOpen)
-          }}
+      <Select
+        labelId="lead-investigator-drop-down"
+        id="lead-investigator-drop-down"
+        value={currentInvestigatorSelected || 'Select pricinple investigator'}
+        onChange={e => {
+          onChange(e.target.value)
+        }}
+        className={classes.container}
+        disableUnderline
+        classes={{
+          selectMenu: classes.selectMenu,
+          root: classes.selectPrincipleInvestigatorButton,
+        }}
+        MenuProps={{
+          classes: { list: classes.listPadding, paper: classes.listBorder },
+          getContentAnchorEl: null,
+          anchorOrigin: {
+            vertical: 'bottom',
+            horizontal: 'center',
+          },
+          transformOrigin: {
+            vertical: 'top',
+            horizontal: 'center',
+          },
+        }}
+      >
+        <MenuItem
+          value="Select pricinple investigator"
```

Comment:
spelling? Also, what happens when nothing is selected? We want to send '"Select pricinple investigator" to the parent control.

Reply:
Ah, i assumed if nothing was selected we wanted to send an empty string to the parent control. I will fix that.